modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/EditKBArticleDisplayContext.java

```diff
@@ -107,6 +107,12 @@ public Map<String, String> getAvailableKBArticleSections()
 	}
 
 	public String getCancelURL() {
+		long resourcePrimKey = getResourcePrimKey();
```

Comment:
this should be 

if FF > return getRedirect

after that, maybe to check the resourcePrimKey, we should check with or without the FF